### PR TITLE
pymssql v2.3.0

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #42

### CODE REVIEW

**Comment by xylar** [comment]
Trying to fix issues in #42.  (New PR because I'm not a maintainer.)

I need this release for https://github.com/conda-forge/apache-airflow-providers-microsoft-mssql-feedstock/pull/23

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -67,3 +64,4 @@ extra:
   recipe-maintainers:
     - primozgodec
     - mariusvniekerk
+    - xylar
```
Comment (xylar) [inline recipe/meta.yaml:67]: I certainly don't need to be a maintainer but I thought it would be friendly to add myself in case I can be helpful.

**Comment by xylar** [comment]
@PrimozGodec and @mariusvniekerk let's see what the tests show but I'm optimistic that I have this working.  Let me know if you'd like to have me as a maintainer.  If not, I'll be glad to remove that commit.

**Review by mariusvniekerk — APPROVED** [review]

**Comment by xylar** [comment]
Thanks @mariusvniekerk!

**Pull request merged**